MedicalDocManagment.WebUI/app/admin/home/home.component.ts

```diff
@@ -1,20 +1,20 @@
-﻿import { Component, OnInit } from '@angular/core';
+﻿import { Component, OnInit, Inject, Injectable } from '@angular/core';
+import { DOCUMENT } from '@angular/platform-browser';
 
 @Component({
     moduleId: module.id,
     templateUrl: 'home.component.html',
 })
-
+@Injectable()
 export class HomeComponent implements OnInit {
 
-    constructor() { }
+    constructor(@Inject(DOCUMENT) private document: any) { }
 
     ngOnInit() {
```

Comment:
Consider removing empty ngOnInit() with accompanying importing OnInit from '@angular/core' and declaring for implementation of OnInit in declaration of class HomeComponent.